### ISSUE TITLE
Version 0.13.2 isn't installing currently

### ISSUE THREAD

**Issue by epicserve** [fabiocaccamo/django-maintenance-mode#60]
When trying to install 0.13.2, I'm getting the following:

```
$ pip install django-maintenance-mode==0.13.2
Collecting django-maintenance-mode==0.13.2
  ERROR: Could not find a version that satisfies the requirement django-maintenance-mode==0.13.2 (from versions: 0.3.1, 0.3.2, 0.3.3, 0.3.4, 0.3.5, 0.4.0, 0.4.1, 0.4.2, 0.4.3, 0.4.4, 0.4.5, 0.4.6, 0.5.0, 0.5.1, 0.5.2, 0.6.0, 0.6.1, 0.6.2, 0.6.3, 0.6.5, 0.6.6, 0.6.7, 0.7.0, 0.7.1, 0.7.2, 0.8.0, 0.9.0, 0.9.1, 0.10.0, 0.11.0, 0.12.0, 0.13.0, 0.13.1)
ERROR: No matching distribution found for django-maintenance-mode==0.13.2
```

**Comment by fabiocaccamo** [comment]
@epicserve I just tried to install the latest version and I confirm the error. :(
I tried to re-upload the file but I receive this error: `HTTPError: 400 Client Error: File already exists.`
I checked on pypi, the release has been uploaded, but only the wheel.
I just released `0.13.3` version, it should be ok.

**Comment by epicserve** [comment]
@fabiocaccamo, thank you for the quick fix. It seems to be working now with version 0.13.3.
